### ISSUE TITLE
[Bridges] bug with multiple variable bounds

### ISSUE THREAD

**Issue by odow** [jump-dev/MathOptInterface.jl#2375]
x-ref #2357

```julia
julia> import MathOptInterface as MOI

julia> include("test/Bridges/sdpa_models.jl")

julia> model =
           MOI.instantiate(GeometricSDPAModel{Float64}; with_bridge_type = Float64);

julia> x = MOI.add_variable(model)
MOI.VariableIndex(1)

julia> set2 = MOI.GreaterThan(0.0)
MathOptInterface.GreaterThan{Float64}(0.0)

julia> set1  = MOI.EqualTo(0.0)
MathOptInterface.EqualTo{Float64}(0.0)

julia> ci = MOI.add_constraint(model, x, set1)
MathOptInterface.ConstraintIndex{MathOptInterface.VariableIndex, MathOptInterface.EqualTo{Float64}}(1)

julia> print(model)
Feasibility

Subject to:

VariableIndex-in-EqualTo{Float64}
 v[1] == 0.0

julia> ci = MOI.add_constraint(model, x, set1)
MathOptInterface.ConstraintIndex{MathOptInterface.VariableIndex, MathOptInterface.EqualTo{Float64}}(1)

julia> print(model)
Feasibility

Subject to:

VectorAffineFunction{Float64}-in-Zeros
 ┌               ┐
 │-0.0 + 1.0 v[1]│
 └               ┘ ∈ Zeros(1)

VariableIndex-in-EqualTo{Float64}
 v[1] == 0.0

julia> MOI.delete(model, ci)
Dict{MathOptInterface.ConstraintIndex, String}()

julia> print(model)
Feasibility

Subject to:

VectorAffineFunction{Float64}-in-Zeros
 ┌               ┐
 │-0.0 + 1.0 v[1]│
 └               ┘ ∈ Zeros(1)
```

**Comment by blegat** [comment]
Also, `MOI.delete` shouldn't return a `Dict`, it should return `nothing`

**Comment by odow** [comment]
Yip. Fixed in #2357
